x-pack/plugins/observability_solution/observability/common/locators/paths.ts

```diff
@@ -15,36 +15,22 @@ export const EXPLORATORY_VIEW_PATH = '/exploratory-view' as const; // has been m
 export const RULES_PATH = '/alerts/rules' as const;
 export const RULES_LOGS_PATH = '/alerts/rules/logs' as const;
 export const RULE_DETAIL_PATH = '/alerts/rules/:ruleId' as const;
-export const SLOS_PATH = '/slos' as const;
-export const SLOS_WELCOME_PATH = '/slos/welcome' as const;
-export const SLO_DETAIL_PATH = '/slos/:sloId' as const;
-export const SLO_CREATE_PATH = '/slos/create' as const;
-export const SLO_EDIT_PATH = '/slos/edit/:sloId' as const;
-export const SLOS_OUTDATED_DEFINITIONS_PATH = '/slos/outdated-definitions' as const;
 export const CASES_PATH = '/cases' as const;
 
+// // SLOs have been moved to its own app (slo). Keeping around for redirecting purposes.
+export const OLD_SLOS_PATH = '/slos' as const;
```

Comment:
@CoenWarmer I saw this [PR](https://github.com/elastic/kibana/pull/153941/files) of yours regarding redirecting `app/observability/exploratory-view` to `app/exploratory-view` and I did similar thing here. I prefixed with `OLD`, since for example new `SLO_DETAIL_PATH` ( '/:sloId' vs '/slos/:sloId') is used from some observability alerting page and I wanted it in this file as well. I want your thoughts on following:

- I wanted to add a redirect to the old slo detail path and edit path, but I din't know how to do it. I tried something like this `<SimpleRedirect to="/:sloId" redirectToApp="slo" />;`, but it didn't work out. Any suggestion how to make it work?
- Since SLO_DETAIL_PATH is used within observability app in these files [use_get_alert_flyout_components.tsx](x-pack/plugins/observability_solution/observability/public/components/alerts_flyout/use_get_alert_flyout_components.tsx) and [alert_actions.tsx](x-pack/plugins/observability_solution/observability/public/pages/alerts/components/alert_actions.tsx), I defined this path here but it is duplicate in this new [file](x-pack/plugins/observability_solution/slo/common/locators/paths.ts) as well, where I have defined all the new paths. I don't like the duplication, but somehow I feel like the new paths need to be inside the new slo plugin. How do you advise me to approach this? Maybe move all SLO paths (old and new) inside observability_shared?